### PR TITLE
feat: reset password ui and functionality

### PR DESCRIPTION
**Pull request by sshanzel**
### Describe what this PR does
- Kind of existed already since I used this flow to test the privileged session.
- Applied the new UI for reset password.

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-332 #done

### CODE REVIEW

File: packages/webapp/pages/account/security.tsx [hunk 1]
```diff
@@ -58,9 +38,14 @@ const AccountSecurityPage = (): ReactElement => {
   } = usePrivilegedSession();
 
   const { mutateAsync: resetPassword } = useMutation(
-    (params: ValidateResetPassword) => submitKratosFlow(params),
+    ({
+      onPasswordReset: _,
```
Comment (rebelchris) [inline packages/webapp/pages/account/security.tsx:42]: Why the _ deconstruct here?

**Review by rebelchris — APPROVED** [review]
Small question, otherwise looks good.

File: packages/webapp/pages/account/security.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/webapp/pages/account/security.tsx:42]: Just so we don't pass the parameter to the actual endpoint, and so the linting won't shout of the variable being unused.

Comment (sshanzel) [reply packages/webapp/pages/account/security.tsx]: Seems like we don't need the `_` and lint doesn't seem to react to it.

**Pull request merged**